src/main/java/fr/sncf/osrd/utils/Interpolation.java

```diff
@@ -0,0 +1,32 @@
+package fr.sncf.osrd.utils;
+
+import java.util.NavigableMap;
+
+public class Interpolation {
```

Comment:
This is most likely a mistake when fixing conflicts. This file shouldn't exist anymore, it has been replaced by `SortedDoubleMap`.